espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/PointerEventHandler.kt

```diff
@@ -109,17 +109,15 @@ class PointerEventHandler(private val touchType: TouchType) : RequestHandler<Mot
         var startX = displayMetrics.widthPixels / 2 - params.x / 2
         var startY = displayMetrics.heightPixels / 2 - params.y / 2
 
-        params.elementId?.let {
+        params.targetElement?.let {
             val view = Element.getViewById(it)
             val viewElement = ViewElement(view)
             startX = viewElement.bounds.left.toLong()
             startY = viewElement.bounds.top.toLong()
         }
 
         // Do down event
-        val downParams = MotionEventParams()
-        downParams.x = startX
-        downParams.y = startY
+        val downParams = MotionEventParams(startX, startY)
```

Comment:
is that expected that some operations were removed?

Reply:
I replaced the setters with a constructor